src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -312,6 +312,14 @@ export const topNavStrings = {
       defaultMessage: 'Save as a new dashboard',
     }),
   },
+  discardChanges: {
+    label: i18n.translate('dashboard.topNave.discardChangesButtonAriaLabel', {
+      defaultMessage: 'Discard changes',
```

Comment:
After talking with Gail, we decided to go with "Discard changes" for the button to keep things consistent with Maps:

![Screenshot 2023-04-13 at 4 41 29 PM](https://user-images.githubusercontent.com/8698078/231898539-377d27bf-7f5f-4633-88e0-30b41990e996.png)